lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -54,21 +57,25 @@ def define
         option :no
 
         precalculate :exempt_benefits do
-          config.exempt_benefits
+          config.exempt_benefits(chosen_cap.to_sym)
```

Comment:
This block was changed into calculate block in the later commit and re-written (including removing the to_sym).